cypress/integration/e2e/commercial.spec.js

```diff
@@ -36,9 +36,12 @@ describe('Commercial E2E tests', function () {
             runLongReadTestFor(`Article?url=${longReadURL}`);
         });
 
-        it(`It should check slots for a long article in Frontend`, function () {
-            Cypress.config('baseUrl', '');
-            runLongReadTestFor(`${longReadURL}?dcr=false`);
-        });
+        // Uncomment to check Locally.
+        // It seems overriding the baseURL works fine locally but hangs on CI.
+
+        // it(`It should check slots for a long article in Frontend`, function () {
+        //     Cypress.config('baseUrl', '');
+        //     runLongReadTestFor(`${longReadURL}?dcr=false`);
+        // });
```

Comment:
You can use Jest.skipOn &rarr; https://github.com/guardian/consent-management-platform/blob/95d97ecb13c45158bf9b1fae0cc284eb588653e1/cypress/integration/sourcepoint-tcfv2.spec.js#L41

Reply:
Ooos I saw that after I merged! Let me do that on a subsequent PR